site/tests/app/controllers/admin/PlagiarismControllerTester.php

```diff
@@ -0,0 +1,34 @@
+<?php namespace tests\app\controllers\admin;
+
+use app\libraries\Core;
+use tests\BaseUnitTest;
+use app\controllers\admin\PlagiarismController;
+
+class PlagiarismControllerTester extends BaseUnitTest
+{
+   /** @var PlagiarismController */
+    protected $controller;
+
+    /** @var Core */
+    protected $core;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->core = $this->createMockCore([], [], [
+            'getGradeableConfig'  => [],
+        ]);
+
+        $this->controller = new PlagiarismController($this->core);
+    }
+
+    /** @test */
+    public function testQueriesMadeInPlagiarismView()
+    {
+        $response = $this->controller->editPlagiarismSavedConfig();
```

Comment:
This function takes an argument, So your test is failing because you need to pass it a gradeable_id. You also probably need to create a file `/var/local/submitty/courses/".$semester."/".$course."/lichen/config/lichen_".$semester."_".$course."_".$gradeable_id.".json"` so that you can trigger the query call.